style.css

```diff
@@ -2646,6 +2646,44 @@ div.comment:first-of-type {
 /*	13. Site Pagination
```

Comment:
The pagination breaks a bit in IE 11, overflowing the right side of the screen. It could be related to `justify-self` and/or `display: initial`, which isn't supported in IE.